<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI::MainWindow</class>
 <widget class="QMainWindow" name="GUI::MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Q-UML</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/img/pic/main.png</normaloff>:/img/pic/main.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionCloseProject"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCreateScope"/>
    <addaction name="separator"/>
    <addaction name="actionAddAssociation"/>
    <addaction name="actionAddDependency"/>
    <addaction name="actionAddGeneralization"/>
    <addaction name="actionAddAggregation"/>
    <addaction name="actionAddComposition"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionProjectsDockWidget"/>
    <addaction name="actionCommandsDockWidget"/>
    <addaction name="actionMessagesDockWidget"/>
    <addaction name="actionElementsDockWidget"/>
    <addaction name="actionProperties"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionAddAssociation"/>
   <addaction name="actionAddDependency"/>
   <addaction name="actionAddGeneralization"/>
   <addaction name="actionAddAggregation"/>
   <addaction name="actionAddComposition"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>&amp;Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>&amp;Save Active Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>&amp;New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionCreateScope">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/pic/icon_action_scope.png</normaloff>:/icons/pic/icon_action_scope.png</iconset>
   </property>
   <property name="text">
    <string>&amp;CreateScope</string>
   </property>
   <property name="toolTip">
    <string>Add new scope to the current project</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/pic/icon_redo.png</normaloff>:/icons/pic/icon_redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo last actions</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/pic/icon_undo.png</normaloff>:/icons/pic/icon_undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo last actions</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionProjectsDockWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Projects</string>
   </property>
   <property name="toolTip">
    <string>Show/hide projects dock widget</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionCommandsDockWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Commands</string>
   </property>
   <property name="toolTip">
    <string>Show/hide commands dock widget</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionMessagesDockWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Messages</string>
   </property>
   <property name="toolTip">
    <string>Show/hide messages dock widget</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="actionElementsDockWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Elements</string>
   </property>
   <property name="toolTip">
    <string>Show/hide scheme elements</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionAddAssociation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/pic/icon_association.png</normaloff>:/icons/pic/icon_association.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add association</string>
   </property>
   <property name="toolTip">
    <string>Add association relation</string>
   </property>
  </action>
  <action name="actionAddDependency">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/pic/icon_dependency.png</normaloff>:/icons/pic/icon_dependency.png</iconset>
   </property>
   <property name="text">
    <string>Add &amp;dependency</string>
   </property>
   <property name="toolTip">
    <string>Add dependency relation</string>
   </property>
  </action>
  <action name="actionAddGeneralization">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/pic/icon_generalization.png</normaloff>:/icons/pic/icon_generalization.png</iconset>
   </property>
   <property name="text">
    <string>Add &amp;generalization</string>
   </property>
   <property name="toolTip">
    <string>Add generalization relation</string>
   </property>
  </action>
  <action name="actionAddAggregation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/pic/icon_aggregation.png</normaloff>:/icons/pic/icon_aggregation.png</iconset>
   </property>
   <property name="text">
    <string>Add aggr&amp;egation</string>
   </property>
   <property name="toolTip">
    <string>Add aggregation relation</string>
   </property>
  </action>
  <action name="actionAddComposition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/pic/icon_composition.png</normaloff>:/icons/pic/icon_composition.png</iconset>
   </property>
   <property name="text">
    <string>Add c&amp;omposition</string>
   </property>
   <property name="toolTip">
    <string>Add composition relation</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="text">
    <string>&amp;Close Active Project</string>
   </property>
   <property name="toolTip">
    <string>Close current project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="toolTip">
    <string>Display Preferences</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>P&amp;roperties</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>GUI::MainWindow</receiver>
   <slot>onExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenProject</sender>
   <signal>triggered()</signal>
   <receiver>GUI::MainWindow</receiver>
   <slot>onOpenProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveProject</sender>
   <signal>triggered()</signal>
   <receiver>GUI::MainWindow</receiver>
   <slot>onSaveProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCreateScope</sender>
   <signal>triggered()</signal>
   <receiver>GUI::MainWindow</receiver>
   <slot>onCreateScope()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCloseProject</sender>
   <signal>triggered()</signal>
   <receiver>GUI::MainWindow</receiver>
   <slot>onCloseProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAbout()</slot>
  <slot>onExit()</slot>
  <slot>onNewProject()</slot>
  <slot>onOpenProject()</slot>
  <slot>onSaveProject()</slot>
  <slot>onCreateScope()</slot>
  <slot>onAddAlias()</slot>
  <slot>onAddUnion()</slot>
  <slot>onAddStruct()</slot>
  <slot>onAddClass()</slot>
  <slot>onAddTemplate()</slot>
  <slot>onMakeRelation()</slot>
  <slot>onCloseProject()</slot>
 </slots>
</ui>
